Bug: Fifo list command spits out error

Tried running fifo list to see if I forgot to add a task when it failed to run today, instead of getting a list of my tasks and triggers I got this:

`[2020-09-22 18:41:39] [ERROR] red: Exception in command 'fifo list'
Traceback (most recent call last):
  File "/home/Sad/redenv/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/Sad/.local/share/Red-DiscordBot/data/red/cogs/CogManager/cogs/fifo/fifo.py", line 309, in fifo_list
    await ctx.maybe_send_embed(out)
  File "/home/Sad/redenv/lib/python3.8/site-packages/redbot/core/commands/context.py", line 254, in maybe_send_embed
    return await self.send(
  File "/home/Sad/redenv/lib/python3.8/site-packages/redbot/core/commands/context.py", line 93, in send
    return await super().send(content=content, **kwargs)
  File "/home/Sad/redenv/lib/python3.8/site-packages/discord/abc.py", line 890, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed,
  File "/home/Sad/redenv/lib/python3.8/site-packages/discord/http.py", line 245, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In embed.description: Must be 2048 or fewer in length.`

--- a/fifo/fifo.py
+++ b/fifo/fifo.py
@@ -10,6 +10,7 @@ from apscheduler.schedulers.base import STATE_PAUSED, STATE_RUNNING
 from redbot.core import Config, checks, commands
 from redbot.core.bot import Red
 from redbot.core.commands import TimedeltaConverter
+from redbot.core.utils.chat_formatting import pagify
 
 from .datetime_cron_converters import CronConverter, DatetimeConverter
 from .task import Task
@@ -306,7 +307,11 @@ class FIFO(commands.Cog):
                 out += f"{task_name}: {task_data}\n"
 
             if out:
-                await ctx.maybe_send_embed(out)
+                if len(out) > 2000:
+                    for page in pagify(out):
+                        await ctx.maybe_send_embed(page)
+                else:
+                    await ctx.maybe_send_embed(out)
             else:
                 await ctx.maybe_send_embed("No tasks to list")
 

